03.image_classification/README.cn.md

```diff
@@ -403,66 +378,76 @@ cost_ploter = Ploter(train_title, test_title)
 step = 0
 def event_handler_plot(event):
     global step
-    if isinstance(event, paddle.event.EndIteration):
+    if isinstance(event, fluid.EndStepEvent):
         if step % 1 == 0:
-            cost_ploter.append(train_title, step, event.cost)
+            cost_ploter.append(train_title, step, event.metrics[0])
             cost_ploter.plot()
         step += 1
-    if isinstance(event, paddle.event.EndPass):
+    if isinstance(event, fluid.EndEpochEvent):
```

Comment:
just curious at the last step of an epoch, will Fluid trigger both `EndStepEvent` and `EndEpochEvent` ?

Reply:
I believe the order is 
EndStepEvent, and then EndEpochEvent